packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -883,4 +889,55 @@ describeCompat("SingleCommit Summaries Tests", "NoCompat", (getTestObjectProvide
 			"Summary Parent should match handle of summary2",
 		);
 	});
+
+	it("enable_single_commit_summary enables Single commit summary in r11s", async function () {
```

Comment:
Test times out when running against FRS in the attach step. Might have to check in the changes without test.